chirp/drivers/icf.py

```diff
@@ -505,6 +505,9 @@ def _clone_to_radio(radio):
             LOG.debug("Waiting for clone result...")
             time.sleep(0.5)
 
+    if radio.ignore_clone_result() == True:
+        return bytes(b'\x00')
```

Comment:
This method returns a boolean, so please don't break that here. Just `return True` here please.

Reply:
Done